tests/Baseline/hilti.optimization.const/opt.hlt

```diff
@@ -0,0 +1,23 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+module Foo {
+
+import hilti;
+
+global auto t = True;
+global auto f = False;
+
+hilti::print(True, True);
+hilti::print(False, True);
+
+if ( True ) {
+    0;
+}
+else {
+    1;
+}
```

Comment:
This don't look optimized (just the constant is inlined)?

Reply:
Like discussed offline, this is related to the patch you reviewed being unable to remove dead `else` branches (there was a `TODO` for that in the code). I now added that ability and this block gets optimized to a single block `{ 0; }`.